src/standalone/Soil/energy_hydrology.jl

```diff
@@ -73,7 +75,8 @@ function EnergyHydrologyParameters{FT}(;
     K_sat::FT,
     S_s::FT,
     θ_r::FT,
-    albedo = FT(0.2),
+    PAR_albedo = FT(0.2),
+    NIR_albedo = FT(0.2),
```

Comment:
Is this surface albedo or soil albedo?

Reply:
soil. It is stored in SoilParameters, so I think the naming is OK

Reply:
So it should be something like: 
SW_albedo = 0.5 * PAR_albedo + 0.5 * NIR_albedo
PAR_albedo = FT(0.2);
NIR_albedo = FT(0.4);